components/dydx_order_manager.py

```diff
@@ -277,6 +278,112 @@ def withdraw_from_dydx(self, recipient_address, dydx_client):
         except Exception as e:
             raise Exception(e)
 
+    #    open order on dydx celery task algo
+    #     asset type -- ETH-USD,BTC-USD etc
+    def open_order_on_dydx(self, order_details):
+        dydx_asset_instance = asset_dydx_instance[order_details["asset_pair"]]
+        dydx_order_manager = DydxOrderManager(dydx_asset_instance)
+        asset_open_position = dydx_order_manager.position_status(
+            "position_status", order_details["binance_asset_pair"]
+        )
```

Comment:
what does this give you ?

Reply:
it return the  true if the  asset position is open on dydx else false

Reply:
oh then can you rename it to `is_positiion_open`